tabcmd/commands/datasources_and_workbooks/datasources_and_workbooks_command.py

```diff
@@ -111,19 +120,23 @@ def is_truthy(value: str):
         return value.lower() in ["yes", "y", "1", "true"]
 
     @staticmethod
-    def apply_png_options(request_options: TSC.ImageRequestOptions, args, logger):
-        if args.height or args.width:
-            # only applicable for png
-            logger.warn("Height/width arguments not yet implemented in export")
+    def apply_png_options(logger, request_options: TSC.ImageRequestOptions, args):
+        try:
+            if args.height or args.width:
+                # only applicable for png
+                logger.warn("Height/width arguments not yet implemented in export")
+        except AttributeError as ae:
+            logger.debug("Unexpected: no user or default value is present: {}".format(ae.__str__()))
```

Comment:
Is this messaging correct? What is `user` in this context?

Reply:
I don't expect to get here because argparse should set a default value for these options. The message means there is no user-set value or default value

Reply:
Ooooh, gotcha. English is hard. I read it as `"There is no user" || "There is no default value"` :) Maybe change `user` to `user-specified`?

Reply:
I ended up just deleting it - I realized that if they aren't set, I would have hit this error well before we get here anyway!